ext/DashBasePlotlyBaseExt.jl

```diff
@@ -0,0 +1,13 @@
+module DashBasePlotlyBaseExt
+
+import DashBase
+import PlotlyBase
+import PlotlyBase.JSON
```

Comment:
This would of course break if PlotlyBase stops importing `JSON`.

I think this here is fine for now, but we should probably make the switch to using `JSON3` exclusively inside DashBase.jl and Dash.jl. We should open an issue once this PR is merged.

Looks like PlotlyBase does define JSON3-compatible `StructType`s https://github.com/JuliaPlots/PlotlyBase.jl/blob/master/src/json3.jl, but I haven't tested it